src/shogun/util/factory.h

```diff
@@ -66,6 +75,11 @@ namespace shogun
 	BASE_CLASS_FACTORY(CSplittingStrategy, splitting_strategy)
 	BASE_CLASS_FACTORY(CMachineEvaluation, machine_evaluation)
 	BASE_CLASS_FACTORY(CSVM, svm)
+	BASE_CLASS_FACTORY(CFeatures, features)
+	BASE_CLASS_FACTORY(CDifferentiableFunction, differentiable_function)
+	BASE_CLASS_FACTORY(CLikelihoodModel, likelihood_model)
+	BASE_CLASS_FACTORY(CMeanFunction, mean_function)
+	BASE_CLASS_FACTORY(CInference, inference)
```

Comment:
@karlnapf I feel that the list of factories is expanding :)

Reply:
yes, but that is fine for now I think